tests/integration/components/frost-scroll-test.js

```diff
@@ -23,58 +14,90 @@ describeComponent(
     integration: true
   },
   function () {
-    let sandbox
+    it('sets full class correctly', function () {
```

Comment:
Not that I'm saying this hurts, but since we're not proxying the class this is essentially testing the ember component interface